MODULE.bazel

```diff
@@ -6,11 +6,19 @@ module(
     compatibility_level = 1,
 )
 
-bazel_dep(name = "aspect_bazel_lib", version = "1.20.0")
+bazel_dep(name = "aspect_bazel_lib", version = "1.21.4")
 bazel_dep(name = "bazel_skylib", version = "1.3.0")
 bazel_dep(name = "rules_nodejs", version = "5.5.3")
 bazel_dep(name = "platforms", version = "0.0.4")
 
+bazel_lib_ext = use_extension("@aspect_bazel_lib//lib:extensions.bzl", "ext")
+
+use_repo(bazel_lib_ext, "copy_to_directory_toolchains")
+
+register_toolchains(
+    "@copy_to_directory_toolchains//:all",
+)
+
```

Comment:
`npm_package` makes use of the toolchain.

Reply:
This should unnecessary after https://github.com/aspect-build/bazel-lib/pull/336?

Reply:
https://github.com/bazelbuild/bazel-central-registry/pull/372 will have what you need here

Reply:
I'll land the version bump in rules_js here https://github.com/aspect-build/rules_js/pull/789